src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/HealthChecks/HealthCheckFunction.cs

```diff
@@ -1,27 +1,30 @@
 ﻿using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Dtos;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Strategies;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.Functions.Worker;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks;
 
-public class HealthCheckFunction
+public class HealthCheckFunction(IEnumerable<IHeathCheckStrategy> strategies)
 {
     [Function(nameof(HealthCheck))]
     [Produces("application/json")]
-    public IActionResult HealthCheck(
-#pragma warning disable IDE0060
+    public async Task<IActionResult> HealthCheck(
         [HttpTrigger(AuthorizationLevel.Anonymous, "get")]
-        HttpRequest request)
-#pragma warning restore IDE0060
+        HttpRequest _,
+        CancellationToken cancellationToken)
```

Comment:
I'm pretty sure we've got this here for a reason, i.e. that `_` causes a binding error with the HTTP error, so I think you'll need to restore this.